src/platforms/hubspot/actions/meetings/create.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from 'zod';
+import { action } from '@/sdk';
+import client from '../../client';
+
+import { transformMeeting } from '@/platforms/hubspot/actions/mappers';
+
+export default action(
+  'meetings-create',
+  {
+    operation: 'create',
+    resource: 'meetings',
+    mutation: true,
+    schema: z.object({
+      hsTimestamp: z.string(),
+      hsMeetingTitle: z.string(),
+      hsMeetingBody: z.string(),
+      hsMeetingLocation: z.string(),
+      hsMeetingStartTime: z.string(),
+      hsMeetingEndTime: z.string(),
+      hubspotOwnerId: z.string(),
+    }),
+    scopes: [],
```

Comment:
is this intentionally empty or is it just unknown?

Reply:
Noticed we didn't have any scopes for these objects in monorepo. My guess is that engagements objects come for free?